provision/kubernetes/deploy.go

```diff
@@ -1413,6 +1417,11 @@ func (m *serviceManager) ensureServices(ctx context.Context, a provision.App, pr
 		if err != nil {
 			return errors.WithMessage(err, "could not to parse base services annotations")
 		}
+		if annotations == nil {
+			annotations = make(map[string]string)
+		}
+		annotations[backendConfigKey] = fmt.Sprintf("{\"default\":\"%s\"}", backendConfigNameForApp(a, process))
```

Comment:
@arthurcgc we could inject this `backendConfigKey` just when backendConfig CRD exists, right ?

Reply:
@wpjunior I mean we could... but it would mean another call to the k8s api just to set the annotation... what do you think @cezarsa ?

Reply:
the output of `ensureBackendConfig` could return this @arthurcgc